js_modules/dagster-ui/packages/ui-core/src/assets/insights/AssetCatalogInsightsLineChart.tsx

```diff
@@ -195,6 +207,41 @@ export const AssetCatalogInsightsLineChart = React.memo(
       }),
       [renderTooltipFn, rgbColors],
     );
+
+    const chartRef = useRef(null);
+    const onClick = (event: any) => {
+      if (chartRef.current) {
+        const chart = ChartJS.getChart(chartRef.current);
+        if (!chart) {
+          return;
+        }
+
+        const clickedElements = chart.getElementsAtEventForMode(
+          event,
+          'y',
+          {axis: 'x', intersect: false},
+          true,
+        );
+        if (clickedElements.length > 0) {
+          const index = clickedElements[0]!.index;
```

Comment:
You can avoid the `!` by checking the truthiness of the element itself, then using that.

```
const element = clickedElements[0];
if (element) {
  ...
}
```